scripts/zones/Attohwa_Chasm/Zone.lua

```diff
@@ -85,6 +85,21 @@ zone_object.onGameHour = function(zone)
         starting at ID.npc.MIASMA_OFFSET. some are supposed to toggle open, but need retail test
         to determine which.  for now, they're just statically set per npc_list.animation
     --]]
+
+    -- Don't allow Citipati or Xolotl to spawn outside of night
```

Comment:
suggestion: 
`-- Don't allow Citipati or Xolotl to spawn outside of night if unclaimed`

if he's claimed he'll stay up until killed or claim is lost. While I believe your code as written allows for that (as disallow doesn't depop mid fight iirc), it's worth mentioning in the comment.